pmd-doc/src/main/java/net/sourceforge/pmd/docs/RuleDocGenerator.java

```diff
@@ -391,7 +391,7 @@ private void generateRuleSetIndex(Map<Language, List<RuleSet>> rulesets) throws
 
                     if (rule instanceof XPathRule || rule instanceof RuleReference && ((RuleReference) rule).getRule() instanceof XPathRule) {
                         lines.add("**This rule is defined by the following XPath expression:**");
-                        lines.add("```");
+                        lines.add("```xpath");
```

Comment:
For the other languages, I used a space before the language hint... But it actually doesn't matter...